### DISCUSSION TITLE
Which information should the correspondent field store exactly?

### DISCUSSION THREAD

**Discussion by mirisbowring** [jonaswinkler/paperless-ng#670] [Support]
Since one only can store one Value per document.

Should I map the Sender? e.X. Amazon.com?
Or should I match the receiver, whom the document belongs to?

For me, the second option would be difficult to widely adopt since many of the letters are addressed to me and my wife combined.

How are you handling this?

**Comment by jonaswinkler** [comment]
From the documentation

> The correspondent of a document is the person, institution or company that a document either originates form, or is sent to.

Depends on whether the document is something you received or sent to someone.

* For incoming invoices, this usually is the name of the company.
* For incoming letters, the name of the person or company who wrote the letter to.
* For outgoing letters, the recipient.
